### DISCUSSION TITLE
How to register CreatedDate in local time zone

### DISCUSSION THREAD

**Discussion by Kazunari001** [Avanade/Beef#269] [Q&A] [answered]
Hi team!

I am using `SqlServer` for `datasource`.
I am having a trouble because `CreatedDate` automatically generated in a database project is added with UTC time instead of local time.
Since the time zone of `SqlServer` is local time, I am guessing that it is a problem with configuration of Beef.
Is there a way to register `CreatedDate` with local time in Beef?

Thank you.

**Comment by chullybun** [comment, marked as answer]
Hi @Kazunari001,

Yes, make the following change in your `Startup.cs` project:

``` csharp
public void ConfigureServices(IServiceCollection services)
{
    Cleaner.DefaultDateTimeTransform = DateTimeTransform.DateTimeLocal;
```

The `Cleaner` class and corresponding `Clean` methods are used extensively within `CoreEx` as a means to manage/transform the `DateTime`. The code above indicates that the transformation is to result in a _Local_ [kind](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.kind) versus the default of _UTC_.

Cheers...

**Reply by Kazunari001** [reply]
Hi @chullybun ,
Thank you so much for your detailed response.👍
I understand that the `Cleaner` class transform to the local timezone.
I also confirmed that the `CreatedDate` value is transform to local timezone during debug.
